_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -106,43 +106,52 @@ Mathematical symbols cannot be substituted for text as [explicitly mentioned in
 
 #### Inapplicable Example 1
 
-Not a widget role.
+This `nav` is not a widget, so the [visible][] text does not need to match the [accessible name][].
 
 ```html
-<a aria-label="OK">Next</a>
+<nav aria-label="main nav">W3C navigation</nav>
 ```
 
 #### Inapplicable Example 2
 
-Widget role that does not support name from content.
+This text field does not need to have its [visible][] text match the [accessible name][]. The content of a textfield shows its value instead of its label. The label is usually adjecent to the textfield instead.
```

Comment:
```suggestion
This text field does not need to have its [visible][] text match the [accessible name][]. The content of a textfield shows its value instead of its label. The label is usually adjacent to the textfield instead.
```